src/ngMock/angular-mocks.js

```diff
@@ -1788,24 +1788,29 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
    * @ngdoc method
    * @name $httpBackend#flush
    * @description
-   * Flushes all pending requests using the trained responses.
+   * Flushes pending requests using the trained responses in the order they arrived.
```

Comment:
Ok, I'll do.
(I am not a native speaker so I will be thankful to get any idea about the text.)